### PR TITLE
5.2.1 release notes

### PR DESCRIPTION
**Pull request by promanenko**

### CODE REVIEW

**Review by utkukaratas — APPROVED** [review]
added "* Improve member address entry UI in cluster config creation. [MC-1376]"

File: docs/modules/release-notes/pages/5-2-1.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+= 5.2.1 Release Notes
+:description: These release notes list any new features, enhancements, and fixes that were made between version 5.2.0 and 5.2.1 of Hazelcast Management Center (MC).
+
+{description}
+
+[[enh-521]]
+== Enhancements
+
+* Allowed READ_ONLY role to see WAN Sync page an events . [MC-1802]
```
Comment (Serdaro) [inline docs/modules/release-notes/pages/5-2-1.adoc:9]: I couldn't get the "WAN Sync page an events". Maybe it should be "WAN Sync page and events"?

File: docs/modules/release-notes/pages/5-2-1.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* Restored logging of MC Web Application Address on startup. [MC-1781]
+* Added a configurable Websockert buffer size. [MC-1672]
```
Comment (Serdaro) [inline docs/modules/release-notes/pages/5-2-1.adoc:11]: I am guessing this is "Websocket"

File: docs/modules/release-notes/pages/5-2-1.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+[[fixes-521]]
+== Fixes
+
+* SSL keystore path no longer overridden by the truststore path. [MC-1818]
+* MC no longer fails when 'hazelcast.mc.contextPath' doesn't start from '/'. [MC-1780]
+* MC no longer throws TaskRejectedExceptions after connecting to a cluster. [MC-1779]
+* Map Browser no longer fails the map value is a Protobuf object. [MC-1718]
```
Comment (Serdaro) [inline docs/modules/release-notes/pages/5-2-1.adoc:20]: Maybe ".. no longer fails if the map value ..."

**Review by Serdaro — COMMENTED** [review]
Minor comments.

**Review by Serdaro — APPROVED** [review]

**Pull request merged**